### PR TITLE
Feature/scaffold daohaus connect

### PR DESCRIPTION
**Pull request by jordanlesich**
## GitHub Issue

We should review and merge https://github.com/HausDAO/daohaus-monorepo/pull/134 before merging this 

https://github.com/HausDAO/daohaus-monorepo/issues/118

## Changes
uses Nx generate to scaffold a react-library 

Tested that code can be served to the apps

Screenshot from 2022-05-05 16-12-39

## Packages Added

NA

## Checks

Before making your PR, please check the following:

- [x] Critical lint errors are resolved
- [x] App runs locally
- [x] App builds locally (run the build command for *any impacted package* and check for any errors before the PR)

### CODE REVIEW

**Review by skuhlmann — APPROVED** [review]
My only bit of feedback is to add `-feature` to the name of the package. Good to merge after that.

discussion here:

https://discord.com/channels/709210493549674598/951186621397618708/972144371082338425

File: apps/summon-app/src/app/App.tsx [hunk 1]
```diff
@@ -1,6 +1,9 @@
+import { DaohausConnect } from '@daohaus/daohaus-connect';
 import { HausThemeContext } from '@daohaus/ui';
 import { useContext } from 'react';
 
+console.log(DaohausConnect);
```
Comment (jonathanprozzi) [inline apps/summon-app/src/app/App.tsx:5]: we can likely remove this log

File: libs/daohaus-connect/package.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "name": "@daohaus/daohaus-connect",
```
Comment (jonathanprozzi) [inline libs/daohaus-connect/package.json:2]: if we determine we want to change this to append `-feature` we'd need to update this too

File: libs/daohaus-connect/project.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
+{
+  "root": "libs/daohaus-connect",
```
Comment (jonathanprozzi) [inline libs/daohaus-connect/project.json:2]: see previous note -- flagging where we'd need to update if we decide to change

**Review by jonathanprozzi — APPROVED** [review]
Looks good -- we may need to make some small changes based on the decision we arrive at for the naming conventions

See your note about merging #134 first -- noticing that this builds on top of those changes so we'll want to be sure to do that to avoid conflicts

**Pull request merged**